admin/authentication/providers.md

```diff
@@ -0,0 +1,21 @@
+# Manage authentication providers
+
+Stoplight supports the following authentication providers:
+
+- Username and password
+- LDAP
+- <!-- QUERY: which others? -->
+
+## Add a new provider
+
+1. Select **Add Provider**.
```

Comment:
We need to call out that this is supporting LDAP specifically.   We should mention that this supports active directory and what not. 

@tommymorgan can you add some color here?

Reply:
My understanding is that LDAP is a protocol and AD is a directory database that supports access via LDAP.  So, to keep it simple, I would probably just make a simple mention of both for now since there's no need to get into the weeds.  Something like the suggestion below would let users know that this functionality covers Active Directory.

e.g.
```
## Add a new LDAP/Active Directory provider
```

Reply:
works for me :)